src/QueryEditorForm.tsx

```diff
@@ -42,51 +46,129 @@ export function QueryEditorForm(props: Props) {
 
   return (
     <>
-      <InlineSegmentGroup>
-        <div className="gf-form-group">
-          <h6>Macros</h6>
-          <ResourceSelector
-            onChange={onChange('schema')}
-            fetch={fetchSchemas}
-            value={props.query.schema || null}
-            tooltip="Use the selected schema with the $__schema macro"
-            label={selectors.components.ConfigEditor.schema.input}
-            data-testid={selectors.components.ConfigEditor.schema.testID}
-            labelWidth={11}
-            className="width-12"
-          />
-          <ResourceSelector
-            onChange={onChange('table')}
-            fetch={fetchTables}
-            value={props.query.table || null}
-            dependencies={[props.query.schema]}
-            tooltip="Use the selected table with the $__table macro"
-            label={selectors.components.ConfigEditor.table.input}
-            data-testid={selectors.components.ConfigEditor.table.testID}
-            labelWidth={11}
-            className="width-12"
-          />
-          <ResourceSelector
-            onChange={onChange('column')}
-            fetch={fetchColumns}
-            value={props.query.column || null}
-            dependencies={[props.query.table]}
-            tooltip="Use the selected column with the $__column macro"
-            label={selectors.components.ConfigEditor.column.input}
-            data-testid={selectors.components.ConfigEditor.column.testID}
-            labelWidth={11}
-            className="width-12"
-          />
-          <h6>Frames</h6>
-          <FormatSelect query={props.query} options={SelectableFormatOptions} onChange={props.onChange} />
-          {props.query.format === FormatOptions.TimeSeries && (
-            <FillValueSelect query={props.query} onChange={props.onChange} />
-          )}
-        </div>
-        <div style={{ minWidth: '400px', marginLeft: '10px', flex: 1 }}>
-          <SQLEditor query={props.query} onChange={props.onChange} datasource={props.datasource} />
-        </div>
-      </InlineSegmentGroup>
+      {newFormStylingEnabled ? (
+        <EditorRows>
+          <EditorRow>
+            <EditorFieldGroup>
+              <EditorField
+                width={15}
+                label={selectors.components.ConfigEditor.schema.input}
+                tooltip="Use the selected schema with the $__schema macro"
```

Comment:
For some reason, description prop in EditorField doesn't show up so Im leaving tooltips for now until we can investigate.
https://github.com/grafana/grafana-experimental/issues/89

Reply:
the underlying field component in `@grafana/ui` renders the description in the label only if the label is a string
 https://github.com/grafana/grafana/blob/bf8af608a7bda9a287071a3f4bdbc96d9d3d335f/packages/grafana-ui/src/components/Forms/Field.tsx#L95-L102

but the editor field component passes in a label element so the description doesn't show up.

https://github.com/grafana/grafana-experimental/blob/ace60c4015b0968813fa055593d8aceadef72e11/src/QueryEditor/EditorField.tsx#L26-L39

Reply:
Indeed, thanks for getting to the bottom of this @kevinwcyu! Added this to the ticket :)